pkg/generators/models/testdata/cases/toplevel_object_with_additional_properties/api.yaml

```diff
@@ -0,0 +1,11 @@
+openapi: 3.0.0
+info:
+  version: 0.1.0
+  title: Test
+
+components:
+  schemas:
+    Foo:
+      type: object
+      additionalProperties:
```

Comment:
Can you add a test with a slightly more complicated example, e.g. when it has a `$ref` or a `oneOf`?

Reply:
Here they are!